client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1288,7 +1288,19 @@
     }
 
     function AuthoringTopbarDirective() {
-        return {templateUrl: 'scripts/superdesk-authoring/views/authoring-topbar.html'};
+        return {
+            templateUrl: 'scripts/superdesk-authoring/views/authoring-topbar.html',
+            link: function(scope) {
+                scope.saveDisabled = false;
+                scope.saveTopbar = function(item) {
+                    scope.saveDisabled = true;
+                    return scope.save(item)
+                    .then(function() {
+                        scope.saveDisabled = false;
+                    });
```

Comment:
IMO it's better to clear the flag in the `finally()` handler, because as it currently stands, the `save` functionality would not be re-enabled in the case of an error (e.g. a random request timeout) - unless this is intended?